exposures/info/block-clockwork-info.yaml

```diff
@@ -0,0 +1,21 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-clockwork-php-page
+  namespace: default # use your own labels here
+spec:
+  message: Access deined for  file to users
```

Comment:
Extra white space is there, please correct this in msg

Reply:
extra space pace is removed